tests/test_response_encoding.py

```diff
@@ -1,32 +1,51 @@
 import random
+import re
 
 import pytest
 from django.http import HttpRequest, JsonResponse
 from django.test.client import Client
 from django.urls import path
-from pydantic import BaseModel
+from pydantic import BaseModel, ConfigDict
 from typing_extensions import TypedDict
 
 from django_api_decorator.decorators import api
 from django_api_decorator.openapi import generate_api_spec
 
 
 class MyTypedDict(TypedDict):
-    a: int
+    an_integer: int
 
 
 class MyPydanticModel(BaseModel):
-    a: int
+    an_integer: int
```

Comment:
Changed to this name to clearly illustrate the difference between snake case and camel case.